os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/common/os/tizenrt/osif_tizenrt.c

```diff
@@ -287,15 +287,31 @@ bool osif_task_signal_clear(void *p_handle)
 /****************************************************************************/
 uint32_t osif_lock(void)
 {
-	return irqsave();
+	uint32_t flags = 0U;
+	if (osif_task_context_check() == true)
+	{
+		save_and_cli();
+	}
+	else
+	{
+		dbg("warn: unexpected isr mode\r\n");
+	}
+	return flags;

```

Comment:
Is this return needed? It always returns `0U`.
I think the line 290 and the line 299 are not needed.

Reply:
Because these API is used in RTK BT Lib, the format cannot be change.
So we need to return, even it always return 0.